sandbox/cri/cri.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+set -ex
+
+mkdir /tmp/mnt
+sudo mount /var/lib/firecracker-containerd/runtime/hello-rootfs.ext4 /tmp/mnt
+sudo cp /firecracker-containerd/agent/agent /tmp/mnt/usr/local/bin
+sudo cp $(which runc) /tmp/mnt/usr/local/bin
+sudo cp /firecracker-containerd/sandbox/cri/fc-agent.start /tmp/mnt/etc/local.d
+sudo ln -s /etc/init.d/local /tmp/mnt/etc/runlevels/default/local
+sudo ln -s /etc/init.d/cgroups /tmp/mnt/etc/runlevels/default/cgroups
+sudo umount /tmp/mnt
+naive_snapshotter -address /var/run/firecracker-containerd/snapshotter.sock -path /var/lib/firecracker-snapshotter &> /dev/null &
+pid=$!
+make test-cri
+exec "$@"
+
+kill -9 $pid
```

Comment:
On line 15, you're `exec`ing another process, so execution will never reach this line. You should probably use the shell's `trap` builtin to execute cleanup code upon termination. Also, avoid `kill -9` except in an escalation sort of scenario. State management programs like snapshotters almost certainly are going to want to perform some pre-shutdown housekeeping. (I realize this is an ephemeral container, so if you _really_ want to use `SIGKILL`, at least include a comment stating why it's the right thing to use here.)